aws_lambda_powertools/utilities/trigger/cloud_watch_logs_event.py

```diff
@@ -0,0 +1,91 @@
+import base64
+import json
+import zlib
+from typing import Dict, List, Optional
+
+
+class CloudWatchLogsLogEvent:
+    def __init__(self, log_event: dict):
```

Comment:
I never know whether to use dict or Dict when typing, but should this be Dict[str, Any]?

Reply:
> I never know whether to use dict or Dict when typing, but should this be Dict[str, Any]?

Sure in this case it would be `Dict[str, Any]`, i will update these and others. Most of these `__init__` methods are for internal use (or unit tests)

Reply:
@jplock i updated these and others.